presto-hive/src/main/java/com/facebook/presto/hive/HiveHdfsConfiguration.java

```diff
@@ -25,7 +25,20 @@
 public class HiveHdfsConfiguration
         implements HdfsConfiguration
 {
-    private static final Configuration DEFAULT_CONFIGURATION = new Configuration();
+    private static final Configuration INITIAL_CONFIGURATION;
+
+    static {
+        Configuration.addDefaultResource("hdfs-default.xml");
```

Comment:
We shouldn't be loading these by default. That's why we have the additional HDFS configs option.